docs/concepts/python-versions.md

```diff
@@ -158,6 +158,26 @@ $ uv python install 3.12.6 --preview  # Does not update `python3.12`
 $ uv python install 3.12.8 --preview  # Updates `python3.12` to point to 3.12.8
 ```
 
+## Upgrading Python patch versions
+
+Managed Python minor versions can be upgraded to the latest patch. This is not currently supported
+for PyPy and GraalPy implementations.
```

Comment:
From a user perspective: What does it mean if an interpreter can or cannot be upgraded, how is that different from just uninstalling the old version and installing the new one?

Reply:
Good point, and actually this is misleading. It's just that transparent virtual environment patch upgrades aren't supported for them. Updating